src/css/page_sections/Menu/_Menu.scss

```diff
@@ -0,0 +1,71 @@
+.coa-Menu {
+  @media screen and (min-width: $coa-large-screen) {
+    margin: $coa-space-level6 auto 0;
+  }
+  @media screen and (max-width: $coa-large-screen) {
+    position: absolute;
+    left: -100vw;
+    width: 100vw;
+    transition: 0.5s;
+    top: 0;
+    bottom: 0;
+    background-color: $color-white;
+    z-index: 2;
+  }
+}
+
+.coa-Menu__list {
+  list-style: none;
+  @media screen and (max-width: $coa-large-screen) {
+    padding: 0 1.2rem;
+  }
+  @media screen and (min-width: $coa-large-screen) {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+.coa-Menu__airplane-icon {
+  margin: 2px .5rem -3px;
+}
+
+.coa-Menu--open {
+  left: 0;
+  right: 0;
+}
+
+.coa-Menu__close-btn {
+  background: transparent;
+  padding: 0;
+  margin: $coa-space-level1;
+  width: auto;
+  &:hover, &:active {
+    background: transparent;
+  }
+  svg {
+    margin: $coa-space-level1 $coa-space-level2;
+  }
+}
+
+.coa-Menu__overlay {
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  transition: opacity 0.2s ease-in-out;
+  background: rgba($color-gray-medium, 1);
+  position: absolute;
+  transition: opacity 0.2s ease-in-out, visibility 0.2s ease-in-out;
+  opacity: 0;
+  visibility: hidden;
+}
+
+.coa-Menu__overlay--open {
+  opacity: 0.3;
+  visibility: visible;
+}
+
+.coa-Menu__footer-text {
+  font-size: $h6-font-size;
```

Comment:
I started pulling out uswds variables from our css so we can move away from it. My branch has some independent of uswds font size definitions set.
https://github.com/cityofaustin/janis/blob/fb35ba5e62f834729f833345dfb06d113559e5d1/src/css/base/_variables_coa.scss